File: c1/python/c1.py
grade = int(input("Enter your grade for programming (0-100): "))
if grade == 100:
    print("Congratulations! You got a perfect score!")

if grade >= 90 and grade <=100:
    print("You got an A ")
elif grade >= 80 and grade < 90:
    print("You got a B")
elif grade >= 70 and grade < 80:
    print("You got a C")
elif grade >= 60 and grade < 70:
    print("You got a D")
else:
    print("You got an F")

File: c1/cpp/c1.cpp
#include <iostream>
using namespace std;

int main(int argc, char ** argv)
{
    int grade;

    cout << "Enter your grade for programming (0-100): " << endl;
    cin >> grade;

    if (grade == 100){
	cout << "Congratulations. You got a perfect score." << endl;
    }

    switch(grade){
	case 90 ... 100:
	    cout << "You got an A" << endl;
	    break;
	case 80 ... 89:
	    cout << "You got a B" << endl;
	    break;
	case 70 ... 79:
	    cout << "You got a C" << endl;
	    break;
	case 60 ... 69:
	    cout << "You got a D" << endl;
	    break;
	default:
	    cout << "You got an F" << endl;
    }

    return 0;
}
